docs/conf.py

```diff
@@ -519,9 +522,6 @@ def on_init(app):  # pylint: disable=unused-argument
         # If we are, assemble the path manually
         bin_path = os.path.abspath(os.path.join(sys.prefix, 'bin'))
         apidoc_path = os.path.join(bin_path, apidoc_path)
-    check_call([apidoc_path, '-o', docs_path, os.path.join(root_path, 'openedx_learning'),
-                os.path.join(root_path, 'openedx_learning/migrations')])
```

Comment:
I disabled auto-generation of all API docs here. There's only a manual generation of docs for public APIs and models, via the `/docs/public_apis` dir. The generated docs are usable but not pretty (a lot of formatting issues). But I figure those could be for later.